### PR TITLE
LWDEV-10465 Add IsAlive API

### PR DESCRIPTION
**Pull request by barsukovskii**

### CODE REVIEW

File: src/main/kotlin/com/lykke/trade/volume/monitoring/service/web/dto/MonitoringStatsDto.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.lykke.trade.volume.monitoring.service.web.dto
+
+import io.swagger.annotations.ApiModel
+
+@ApiModel("MonitoringStats")
+class MonitoringStatsDto(@JvmField val VmCpuLoad: Double,
```
Comment (olpapchenko) [inline src/main/kotlin/com/lykke/trade/volume/monitoring/service/web/dto/MonitoringStatsDto.kt]: suggest using standard java naming convention, if we need capital letter in json, we can configure json serialization code to use starting capital letter

File: src/main/kotlin/com/lykke/trade/volume/monitoring/service/isalive/impl/IsAliveResponseGetterImpl.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+import com.lykke.trade.volume.monitoring.service.entity.IsAliveResponse
+import com.lykke.trade.volume.monitoring.service.isalive.IsAliveResponseGetter
+import com.lykke.trade.volume.monitoring.service.monitoring.MonitoringStatsCollector
+import org.apache.http.HttpStatus
+
+class IsAliveResponseGetterImpl(private val version: String,
+                                private val monitoringStatsCollector: MonitoringStatsCollector) : IsAliveResponseGetter {
+    override fun getResponse(): IsAliveResponse {
+        return IsAliveResponse(HttpStatus.SC_OK,
```
Comment (olpapchenko) [inline src/main/kotlin/com/lykke/trade/volume/monitoring/service/isalive/impl/IsAliveResponseGetterImpl.kt]: service layer should not know anything about `http`

File: src/main/kotlin/com/lykke/trade/volume/monitoring/service/web/dto/MonitoringStatsDto.kt (hunk 1 above)
Comment (barsukovskii) [reply src/main/kotlin/com/lykke/trade/volume/monitoring/service/web/dto/MonitoringStatsDto.kt]: fixed

File: src/main/kotlin/com/lykke/trade/volume/monitoring/service/isalive/impl/IsAliveResponseGetterImpl.kt (hunk 2 above)
Comment (barsukovskii) [reply src/main/kotlin/com/lykke/trade/volume/monitoring/service/isalive/impl/IsAliveResponseGetterImpl.kt]: fixed

**Review by olpapchenko — APPROVED** [review]

**Pull request merged**